src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/satsendring/BarnetilsynSatsendringService.kt

```diff
@@ -103,9 +102,11 @@ class BarnetilsynSatsendringService(
 
     @Transactional
     fun opprettTask() {
-        if (LeaderClient.isLeader() == true) {
-            logger.info("Oppretter satsendring-task")
-            val task = BarnetilsynSatsendringTask.opprettTask()
+        val payload = YearMonth.now().toString()
+        val finnesTask = taskService.finnTaskMedPayloadOgType(payload, BarnetilsynSatsendringTask.TYPE)
```

Comment:
Hvordan blir dette hvis begge podene kjører denne samtidig?

Reply:
Tanken var å bruke LeaderClient, men vi "debugger" litt, siden tasken ikke blir opprettet - eller kanskje det er scheduleren som ikke kjører. Hvis to podder kjører så blir det duplikate logginnslag, men vi kan skille på fagsakid i loggen.

Reply:
Tasken blir ikke opprettet, så "debugger" litt for å se om det er scheduleren som ikke kjører. Dette gir vel potensielt en race condition, så skal forsøke å legge på LeaderClient igjen etterpå.

Reply:
La tilbake LeaderClient og logger om scheduler kjøres i stedet.